src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -164,7 +164,7 @@ public static ObjectHandle CreateInstance(string assemblyName, string typeName,
                                                 culture,
```

Comment:
I thought that the null check you are about to delete is the one after `It's classic managed type (not WinRT type)`.

Reply:
Ah...there was a misunderstanding...you copy/paste my comment on `return (o != null) ? new ObjectHandle(o) : null;`. Test case for this coverage found :smile:!
I'll remove this https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Activator.cs#L154!